openapi/golem-service.yaml

```diff
@@ -2901,6 +2917,28 @@ components:
     HealthcheckResponse:
       type: object
     HttpApiDefinition:
+      type: object
+      properties:
+        id:
+          type: string
+        version:
+          type: string
+        routes:
+          type: array
+          items:
+            $ref: '#/components/schemas/Route'
+        draft:
+          type: boolean
+        createdAt:
+          type: string
+          format: date-time
+      required:
+      - id
+      - version
+      - routes
+      - draft
+      - createdAt
```

Comment:
I think this new field needs to be optional to not break the API

Reply:
it is used just in responses, not in requests, 
in such case it can break, if somebody use new client version, but will have old version of server, if we want to avoid that scenario then it had to be optional

Reply:
changed to optional